src/seismometer/controls/selection.py

```diff
@@ -11,26 +11,25 @@ class SelectionListWidget(ValueWidget, VBox):
 
     value = traitlets.Tuple(help="The selected values for the button list")
 
-    def __init__(self, title: str, options: tuple[str], value: tuple[str] = None, show_title: bool = True):
+    def __init__(self, options: tuple[str], *, value: tuple[str] = None, title: Optional[str] = None):
         """A vertical list of buttons for selection
 
         Parameters
         ----------
-        title : str
-            Title to be displayed above the list of buttons.
         options : tuple[str]
             Selectable options.
         value : Optional[tuple[str]], optional
             Subset of options that should be selected by default, by default None.
-        show_title : bool, optional
-            If True, shows the title, by default True.
+        title : Optional[str[], optoinal
+            Title to be displayed above the list of buttons, it not set, title not included
```

Comment:
it not set, title not included -> **if** not set, title not included

Reply:
FINC